bayesian/api/user_api.py

```diff
@@ -0,0 +1,70 @@
+"""Definition of all REST API endpoints for user management."""
+import uuid
+
+from flask import Blueprint, request
+from flask.json import jsonify
+
+from fabric8a_auth.auth import login_required
+from fabric8a_auth.errors import AuthError
+
+from bayesian.exceptions import HTTPError
+from bayesian.utility import user_utils
+from bayesian.utility.user_utils import UserException
+
+user_api = Blueprint('user_api', __name__, url_prefix='/user')
+
+
+@user_api.route('/readiness')
```

Comment:
why do we need readiness and liveness probe here? Its for the pod health check.

Reply:
Will remove it.